packages/eds-lab-react/src/components/SideBar/ActionButton/index.tsx

```diff
@@ -89,39 +89,41 @@ const Tooltip = styled(EDSTooltip)`
   }
 `
 
-export type CreateItemProps = {
-  createLabel: string
-  onCreate: () => void
+export type ActionButtonProps = {
+  actionLabel: string
+  actionIcon: IconData
```

Comment:
Suggest dropping the the pre "action" in wording of prop name.
```suggestion
  label: string
  icon: IconData
```

Reply:
Done